__tests__/app.test.js

```diff
@@ -0,0 +1,47 @@
+const app = require("../app");
+const db = require("../db/connection");
+const request = require("supertest");
+
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("GET /api/topics", () => {
+  test("200: responds with an array of objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        const { body } = response;
+        expect(Array.isArray(body.topics)).toBe(true);
+      });
+  });
+  test("200: responds with array of objects with slug and description properties", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        const { body } = response;
```

Comment:
Just needs a length check on the array to make sure you're getting back all the topics